app/views/setup/_registry.html.slim

```diff
@@ -0,0 +1,15 @@
+= f.fields_for "insecure_registries[]", [] do |field|
+  .row.registry-entry
+    .form-group.col-sm-3
+      = field.text_field :docker_registry_name, placeholder: "Registry name", type: "text", value: (local_assigns[:registry] ? registry.name : "" ), disabled: !local_assigns[:registry].nil?
```

Comment:
Could we pass `registry: DockerRegistry.new` instead of nothing to this partial when there's no registries to avoid this ternary conditions everywhere? Just wondering...